tests/strace--always-show-pid.test

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -efu
+#
+# Check decoding of --argv0 option.
```

Comment:
This test is ecpected to check something else.

Reply:
Fixed in the latest HEAD